server/graphql/v2/object/HostMetricsTimeSeries.ts

```diff
@@ -51,5 +51,14 @@ export const HostMetricsTimeSeries = new GraphQLObjectType({
         return { dateFrom, dateTo, timeUnit, nodes: resultsToAmountWithSettlementNode(results) };
       },
     },
+    totalMoneyManaged: {
+      type: new GraphQLNonNull(TimeSeriesAmount),
+      description: 'History of the total money managed by this host',
+      resolve: async ({ host, dateFrom, dateTo, timeUnit }) => {
```

Comment:
Don't we want the ability to filter by `accounts`?

Reply:
Yes, somehow we've missed this one. I've added support for filter by collective. 👍🏽 

I think we are missing this on the host fee chart as well. I'll open a new issue for that.

Reply:
Actually the host fee chart doesn't need the filtering as its specifically for one year. Sorry I spoke too soon. 😄